vignettes/SEIRD_age_structred_with_interventions.Rmd

```diff
@@ -0,0 +1,136 @@
+---
+title: "SEIRD_age_structred_with_interventions"
+author: "Oscar Dimdore-Miles, Idil Cazimoglu and Liangti Dai"
+date: "28/10/2021"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{The SEIRDAge_interventions model}
+  %\VignetteEngine{knitr::knitr}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(comomodels)
+library(tidyverse)
+```
+
+## Introduction
```

Comment:
I would add to the notebook some extra information, including the equations and the fact that we model self isolation and other interventions by allowing a change in beta.

Reply:
(Notebook modified with information added)